app/domain/authentication/handler/authentication_handler.rb

```diff
@@ -78,7 +78,8 @@ def call(parameters:, request_ip:)
 
         TokenFactory.new.signed_token(
           account: parameters[:account],
-          username: role.role_id.split(':').last
+          username: role.role_id.split(':').last,
+          user_ttl: authenticator.token_ttl
```

Comment:
Is it safe to assume `token_ttl` will always be defined here? I understand this handler class currently only supports the OIDC/pkce authenticators which have been updated accordingly but wasn't sure if the intent was to generalize it for other cases

Reply:
The intent is definitely to extend this pattern to other authenticators, so you're right - currently this is a safe assumption, but that may not always hold true. I think this may be acceptable - as long as we have one flavor of `Authenticator` that has to report a customized `token_ttl`, I'm not sure if this is avoidable.